core/README.md

```diff
@@ -1,4 +1,15 @@
-core
+# core/libdreamcast
```

Comment:
We don't have core/libdreamcast yet. However since it's the way we want to got, it's fine by me.